8Knot/app.py

```diff
@@ -91,7 +91,55 @@
 server = app.server
 server = _login.configure_server_login(server)
 
+"""REGISTER CLIENTSIDE CALLBACKS"""
```

Comment:
What is the purpose of these callbacks? Is there a reason they are implemented differently than everything else?

Reply:
theres a comment that says "# Clientside callback to handle clicking outside the search dropdown", but agreed, id be curious for a little more detail on why this is done this way